frontend/components/ui/blocknote/blocks/quiz-multiple-choice-block.tsx

```
@@ -0,0 +1,218 @@
+  {
+    type: "quiz-multiple-choice",
+    propSchema: {
+      ...defaultProps,
+      question: {
+        default: "",
+      },
+      options: {
+        default: [
+          { id: "1", text: "", isCorrect: true },
+          { id: "2", text: "", isCorrect: false },
+        ] as any,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)